src/commands/utility/assignMeTaskThread.js

```diff
@@ -0,0 +1,82 @@
+const { SlashCommandBuilder } = require('discord.js');
+const { DISCORD_SERVER } = require('../../config');
+const { translateLanguage } = require('../../languages/index');
+
+const ASSIGN_EMOJI = '⚔';
+const ASSIGNED_PATTERN = /\|⚔ @[\w-]+\|/;
```

Comment:
Podriamos reusar la variable al crear el regexp para que sea facil de cambiar en un futuro
```suggestion
const ASSIGN_EMOJI = '⚔';
const ASSIGNED_PATTERN = new RegExp(`${ASSIGN_EMOJI} @[\w-]+\|`)
```